ansible/configs/ocp-workshop/env_vars.yml

```diff
@@ -116,7 +116,7 @@ env_specific_images:
 
 #### Vars for the OpenShift Ansible hosts file
 master_api_port: 443
-ovs_plugin: "subnet" # This can also be set to: "multitenant"
+ovs_plugin: "networkpolicy" # This can also be set to: "multitenant" or "subnet"
```

Comment:
what is the reason to change default here ?

Would this break 3.6 and earlier deployments ?